content/en/data_jobs/airflow.md

```diff
@@ -139,8 +139,11 @@ For Astronomer customers using Astro, <a href=https://www.astronomer.io/docs/lea
    openlineage-python>=1.23.0
    ```
 
-2. Configure the OpenLineage provider. You can do this by setting the following environment variables [using the Astro UI][5]:
-   
+2. To set up the OpenLineage provider, define the following environment variables. You can configure these variables in your Astronomer deployment using either of the following methods:
+
+  - [From the Astro UI][5]: Navigate to your deployment settings and add the environment variables directly.
+  - [In the Dockerfile][11]: Define the environment variables in your `Dockerfile` to ensure they are included during the build process.
+
```

Comment:
Can we add a troubleshooting section at the bottom that calls out setting the env variables in the .env local file doesn't work and to make sure to use the methods we mention?